src/service.rs

```diff
@@ -214,12 +213,16 @@ impl Service {
 
             // Connect to our listening ports, this should unblock
             // the threads.
-            for port in state.listening_ports.iter() {
+            for port in &state.listening_ports {
                 let addr = ::util::loopback_v4(*port).get_address();
 
                 match *port {
-                    Port::Tcp(_) => { let _ = TcpStream::connect(&addr); },
-                    Port::Utp(_) => { let _ = UtpSocket::connect(&addr); }
+                    Port::Tcp(_) => {
+                        let _ = TcpStream::connect(&addr);
+                    }
+                    Port::Utp(_) => {
+                        let _ = UtpSocket::connect(&addr);
+                    }
```

Comment:
Personally I think it is easier to see the differences between the two match statements in the original.